proposals/b:zoe1/submission/upgrade-zoe-permit.json

```diff
@@ -0,0 +1,18 @@
+{
+  "consume": {
+    "vatAdminSvc": "vatAdminSvc",
+    "vatStore": "vatStore",
+    "agoricNamesAdmin": "makeCoreProposalBehavior",
+    "zoe": "makeCoreProposalBehavior"
```

Comment:
I think `makeCoreProposalBehavior` uses `zoe` for `E(zoe).install()` but we're not installing anything in this proposal, so again, this is excess authority. But zoe is designed to be shared widely (modulo #3725, #4395), so not much harm.